packages/airnode-deployer/docker/entrypoint.sh

```diff
@@ -4,4 +4,6 @@ DEPLOYER_BINARY="/usr/local/bin/airnode-deployer"
 USER_ID="${USER_ID:-0}"
 GROUP_ID="${GROUP_ID:-0}"
 
+test -f aws.env && export $(egrep -v '^#' aws.env | xargs)
```

Comment:
Have you tested this?
![Screenshot_20220713_101717](https://user-images.githubusercontent.com/1655405/178685458-4253dcc3-5156-4b9c-b771-90f29f9a5629.png)

Reply:
Yes I tested deploying with both double quotes and without for the AWS credentials (on ubuntu). It doesn't work for you? 

I haven't done anything with the mnemonics since those are handled through the `secrets.env` file, right?

Reply:
Also forgot to mention that I got that export line from Airkeeper (we use it to export credentials there)

Reply:
Yeah, sorry, I haven't realized we're not talking about the secrets file but about the AWS credentials. Because this doesn't work with variables that contain spaces (as you can see in the screenshot). But the AWS credentials don't contain those so I guess it's fine.

Can you maybe change the regex to `'^\s*#'` to allow spaces in front of the `#` sign? I think that's an easy mistake that someone can make and it shouldn't cause problems.

Reply:
Yeah definitely that makes sense, added! :smile: